src/EventStore.Plugins/Licensing/License.cs

```diff
@@ -1,18 +1,39 @@
-﻿using System.Security.Cryptography;
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Security.Cryptography;
 using Microsoft.IdentityModel.JsonWebTokens;
 using Microsoft.IdentityModel.Tokens;
 using static System.Convert;
 
 namespace EventStore.Plugins.Licensing;
 
 public record License(JsonWebToken Token) {
-	public async Task<bool> IsValidAsync(string publicKey) {
+	public string? CurrentCultureIgnoreCase { get; private set; }
+
+	public async Task<bool> ValidateAsync(string publicKey) {
 		var result = await ValidateTokenAsync(publicKey, Token.EncodedToken);
 		return result.IsValid;
 	}
 
-	public bool IsValid(string publicKey) =>
-		IsValidAsync(publicKey).GetAwaiter().GetResult();
+	public bool HasEntitlements(string[] entitlements, [MaybeNullWhen(true)] out string missing) {
+		foreach (var entitlement in entitlements) {
+			if (!HasEntitlement(entitlement)) {
+				missing = entitlement;

```

Comment:
This isn't necessary for the preview release since there is only one entitlement, but it might be more user-friendly to return all of the missing entitlements rather than just the first one

Reply:
yeah fair comment, lets consider that in the future especially if it turns out that some plugins have multiple required entitlements